rocks/RocksCreator.java

```diff
@@ -0,0 +1,52 @@
+package grakn.core.rocks;
+
+import grakn.core.common.parameters.Arguments;
+import grakn.core.common.parameters.Options;
+
+import java.nio.file.Path;
+
+public class RocksCreator {
```

Comment:
`RocksCreator` is responsible for creating "rocks" instances (database manager, database, session, transaction, and storage). It is a "factory" class that can create either Core or Cluster specific rocks instances.

For example, the Core implementation of `RocksCreator` here will return a `RocksDatabase` that is not replicated. The `RockCreator` in Cluster however, will return a `RocksDatabase` that supports replication.

Reply:
Following the factory approach allows for Grakn Cluster to reuse Core's `//rocks` package with minimal modification - only re-creating the "rocks" classes that need to be customised.